src/main/java/rape/brutal/gitstarship/parts/Engine.java

```diff
@@ -10,12 +10,13 @@
  */
 public class Engine extends ShipPart {
 
-    private int cost;
-    private String name;
-    private int size;
-
     private int power;
 
+    public Engine() {
```

Comment:
Power parameter is needed.

You can do it via parameter or via concrete subclasses: `SmallEngine`, `MediumEngine`, `ExtraEngine`.